oscap_report/html_report/templates/rule_detail.html

```diff
@@ -142,19 +142,25 @@
                 </td>
             </tr>
             {% endif %}
-            {% if rule.message != "" %}
+            {% if rule.messages %}
             <tr role="row">
-                <td role="cell"><b>Message:</b></td>
+                <td role="cell"><b>Messages:</b></td>
                 <td role="cell">
+                    {% for message in rule.messages %}
                     <div class="pf-c-alert pf-m-info pf-m-inline">
                         <div class="pf-c-alert__icon">
                             <i class="fas fa-fw fa-info-circle" aria-hidden="true"></i>
                         </div>
                         <p class="pf-c-alert__title">
                             <span class="pf-screen-reader">Message:</span>
-                            {{ rule.message }}
+                            Message:
```

Comment:
I believe the result would be `Message: Message: The text of the message` in this case. Or I'm missing something here?

Reply:
Span with class `pf-screen-reader` is for accessibility reasons. 
Here is the result: 
![image](https://user-images.githubusercontent.com/26072444/153257121-dd614622-3dd0-426c-85cd-2b89bffcf164.png)

Reply:
I don't get it. Why do we need a hidden, screen-reader only `Message:` when we already have the same `Message:`, which is readable by screen reader as well as any other client?

Reply:
Summoning @vojtapolasek.

Reply:
Here is a report example [report.zip](https://github.com/OpenSCAP/openscap-report/files/8035470/report.zip). Try rule: `xccdf_org.ssgproject.content_rule_package_aide_installed`

Reply:
We discussed it with @vojtapolasek and agreed to make a global do-over of a11y at some point.